org.jacoco.core.test/src/org/jacoco/core/instr/ClassFileVersionsTest.java

```diff
@@ -147,8 +148,14 @@ public void visitFrame(int type, int nLocal,
 
 							@Override
 							public void visitEnd() {
-								assertEquals(Boolean.valueOf(expected),
-										Boolean.valueOf(frames));
+								if (CondyProbeArrayStrategy.B_DESC
+										.equals(desc)) {
+									assertEquals(Boolean.FALSE,
```

Comment:
Use `assertFalse()`

Reply:
Add comment:

    // CondyProbeArrayStrategy does not need frames:

Reply:
> Use `assertFalse()`

done

> Add comment

added as assertion message